yt_dlp/cookies.py

```diff
@@ -87,13 +82,13 @@ def load_cookies(cookie_file, browser_specification, ydl):
     return _merge_cookie_jars(cookie_jars)
 
 
-def extract_cookies_from_browser(browser_name, profile=None, logger=YDLLogger()):
+def extract_cookies_from_browser(browser_name, profile=None, keyring=None, logger=YDLLogger()):
```

Comment:
do not change public functions' signature
```suggestion
def extract_cookies_from_browser(browser_name, profile=None, logger=YDLLogger(), keyring=None):
```

Reply:
seems ugly to not have the logger as the last parameter but OK

Reply:
I know, but I don't want to accidentally break anything if someone else was calling these functions. We should have made logger a keyword only argument to avoid this. Maybe we should atleast add the new args as keyword-only
```suggestion
def extract_cookies_from_browser(browser_name, profile=None, logger=YDLLogger(), *, keyring=None):
```